notebook/i18n/de/LC_MESSAGES/notebook.po

```diff
@@ -0,0 +1,416 @@
+# Translations template for PROJECT.
+# Copyright (C) 2017 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2017-02-04 22:21-0600\n"
+"PO-Revision-Date: 2017-02-04 23:04-0600\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Globalization Pipeline\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
+"X-Generator: Poedit 1.8.11\n"
+
+msgid "The Jupyter Notebook requires tornado >= 4.0"
+msgstr "Der Jupyter Notizblock erfordert Tornado >= 4.0"
```

Comment:
I think I'd treat 'Notebook' as part of a name and not translate it. Likewise 'Widgets' a few lines below.